app/assets/stylesheets/modules/sitter.scss

```diff
@@ -2,69 +2,75 @@
 @import "mixins";
 
 //SEARCH
+.main-head {
+  text-align: center;
+  font-family: $dogstay-font-theme;
+  font-size: 3em;
+  padding-bottom: 0.5em;
+  @media only screen and (max-width: 40.063em) { font-size: 1em; };
+}
+
 .all {
+  font-family: $dogstay-font-theme;
   max-width: 100%;
   background: #fff;
-  border: 1px solid #f9f9f9;
 }
-  .container {
-    width: 80%;
-    margin-left: auto;
-    margin-right: auto;
-
-  #sitter-profile-box {
-    margin-right: auto;
-    position: relative;
-    float: left;
-    display: inline-block;
-    margin-top: 1em;
-    margin-bottom: 1em;
+
+.all-sitters {
+  padding-top: 1em;
+
+  .hr {
+    width: 85%;
+    margin-left: 3em;
+  }
+
+  .sitters-profiles {
+    padding-top: 0.5em;
+
 
     img {
-      width: 320px;
-      height: 275px;
+      @include image-style;
+      @media only screen and (max-width: 40.063em) { border-radius: 0; width: 4em; height: 4em; };
     }
 
-    .price {
-      font-family: $sitter-font-type;
-      font-size: 3em;
-      color: #fff;
-      position: absolute;
-      top: 1%;
-      width: 100%;
-      text-align: right;
-      padding-right: 0.3em;
+    .row-name {
+      text-align: center;
+      font-size: 0.9em;
+      @media only screen and (max-width: 40.063em) { font-size: 0.5em; };
     }
 
-    .headline {
-      font-family: $sitter-font-type;
-      font-size: 1.5em;
-      font-weight: bold;
-      color: #fff;
-      position: absolute;
-      padding-left: 0.2em;
-      bottom: 10%;
-      width: 100%;
+    .row-address {
       text-align: center;
-      text-shadow: 1px black;
+      font-size: 0.9em;
+      @media only screen and (max-width: 40.063em) { font-size: 0.5em; };
     }
+  }
 
-    .sitter-name {
-      font-family: $dogstay-font-theme;
-      color: #fff;
-      position: absolute;
-      padding-left: 0.2em;
-      width: 100%;
-      bottom: 2%;
-      text-align: center;
+  .sitters-info {
+    padding-left: 2em;
 
-        a {
-          color: #fff;
-          font-weight: bold;
-        }
+    .sitter-headline {
+      font-family: $sitter-font-type;
+      font-weight: bold;
+      font-size: 1.5em;
+      @media only screen and (max-width: 40.063em) { font-size: 1em; };
+
+      div.sitters-price {
+        float: right;
+        padding-right: 3em;
+        @media only screen and (max-width: 40.063em) { font-size: 1em; };
       }
     }
+
+    .icons {
+      @media only screen and (max-width: 560px) { display: none};
+      img {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4